### DISCUSSION TITLE
How to disable snippet?

### DISCUSSION THREAD

**Discussion by mh-lan** [James-Yu/LaTeX-Workshop#3691] [Q&A] [answered]
Hi everyone, I want to disable snippets of latex workshop, such as `BIT` to begin an itemize environment. 
I have searched the settings.json, but which parameter should I change?

**Comment by jlelong** [comment, marked as answer]
There is currently no setting to disable all snippets from an extension. 

The simplest way is to go to your [extension folder](https://code.visualstudio.com/docs/editor/extension-marketplace#_where-are-extensions-installed), then edit the file `james-yu.latex-workshop-XXX/data/latex-snippet.json` where `XXX` stands for the version number and replace its content with `{}`. Unfortunately, you will have to do this again after every extension update .

Alternatively you can apply the procedure described [here](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_can-i-remove-snippets-from-intellisense) for every single snippet you want to remove.

**Reply by mh-lan** [reply]
Many thanks!
